### PR TITLE
My proofread

### PR DESCRIPTION
**Pull request by trangdata**
TODO: add calibration curve discussion.

### CODE REVIEW

**Review by arielah — APPROVED** [review]
Looks good!

File: content/10.methods.md [hunk 1]
```diff
@@ -72,40 +72,42 @@ We predicted the gender of honorees and authors using the <https://genderize.io>
...
-`ethnicolr` also uses United States Census data, but also adds in data curated from Wikipedia in 2009 [@doi:10.1145/1557019.1557032].
-While ethnicolr overcomes some of the limitations of wru, its international representation is still limited.
+To complement wru's race and ethnicity estimation, we developed a model to predict geographical origins of names.
+The existing Python package ethnicolr [@arxiv:1805.02109] produces reasonable predictions, but its international representation in the data curated from Wikipedia in 2009 [@doi:10.1145/1557019.1557032] is still limited.
 For instance, 76% of the names in ethnicolr's Wikipedia dataset are European in origin, and the dataset contains remarkably fewer Asian, African, and Middle Eastern names compared to that of wru.
-Because the primary weaknesses of ethnicolr were related to the source data, we created an updated training dataset and set of predictors based on the ethnicolr methodology.
 
-#### Constructing a Name-to-Nationality Dataset
+Improving upon ethnicolr, we built a similar classifier, a Long Short-term Memory (LSTM) neural network, to infer the region of origin from patterns in the sequences of letters in full names.
```
Comment (cgreene) [inline content/10.methods.md]:
```suggestion
To address the limitations of ethnicolr, we built a similar classifier, a Long Short-term Memory (LSTM) neural network, to infer the region of origin from patterns in the sequences of letters in full names.
```

File: content/10.methods.md [hunk 2]
```diff
@@ -72,40 +72,42 @@ We predicted the gender of honorees and authors using the <https://genderize.io>
...
+However, we applied this model on an updated, approximately 4.5 times larger training dataset called Wiki2019 (described below).
```
Comment (cgreene) [inline content/10.methods.md]:
```suggestion
We applied this model on an updated, approximately 4.5 times larger training dataset called Wiki2019 (described below).
```

File: content/20.results.md [hunk 3]
```diff
@@ -60,16 +60,17 @@ We found that white honorees have been significantly overrepresented and Asian h
...
 We found similar performance across 1, 2, and 3-grams; however, the classifier required fewer epochs to train with 3-grams so we used this length in the model that we term Wiki2019-LSTM.
-Our Wiki2019-LSTM returns, for each given name, a probability of that name originating from each of our 8 regions.
-We observed a multiclass ROC/AUC score of 0.954 for the classifier, indicating that the classifier can recapitulate name origin with high sensitivity and specificity.
-AUC calculated for each region individually was also high (Fig. {@fig:wiki2019_lstm}), suggesting that our classifier was sufficient for use in a broad-scale examination of disparities.
+Our Wiki2019-LSTM returns, for each given name, a probability of that name originating from each of the specified eight regions.
+We observed a multiclass area under the receiver operating characteristic curve (AUC) score of 95.4% for the classifier, indicating that the classifier can recapitulate name origins with high sensitivity and specificity.
+For each individual region, the high AUC (above 94%, Fig. {@fig:wiki2019_lstm}) suggests that our classifier was sufficient for use in a broad-scale examination of disparities.
```
Comment (cgreene) [inline content/20.results.md:70]:
```suggestion
We also observed that the model was well calibrated (Fig. {@fig:wiki2019_lstm}B).

```

File: content/20.results.md [hunk 4]
```diff
@@ -60,16 +60,17 @@ We found that white honorees have been significantly overrepresented and Asian h
...
```
Comment (cgreene) [inline content/20.results.md]:
```suggestion
For each individual region, the high AUC (above 94%, Fig. {@fig:wiki2019_lstm}A) suggests that our classifier was sufficient for use in a broad-scale examination of disparities.
```

**Review by cgreene — APPROVED** [review]
This looks good to me. I had a few light suggestions but I don't feel strongly about most of them. The only one that I think is absolutely important is referencing the calibration panel within the text itself somewhere.

**Comment by cgreene** [comment]
I want to merge this and give ourselves another length check. Is that ok? I don't want to interrupt if you've got some more changes that you're planning on this branch.

**Pull request merged**